app/jobs/sync_with_blue_alliance_job.rb

```diff
@@ -11,40 +11,35 @@ def event_list(year = Time.now.year.to_s)
 		end
 	end
 
-	def team_list()
-		page = 0
-		teams = []
-		response = self.class.get("/teams/#{page}")
-		while response.success? && !response.parsed_response.empty?
-			teams += response.parsed_response
-			page += 1
-			response = self.class.get("/teams/#{page}")
-		end
-		teams
+	def team_list(event_key)
+		response = self.class.get("/event/#{event_key}/teams/simple")
+		response.parsed_response
 	end
 	
 	def perform()
 		old_logger = ActiveRecord::Base.logger
 		ActiveRecord::Base.logger = nil
 		Team.delete_all
 		Competition.delete_all
+		puts "Downloading events list..."
 		events = event_list()
+		puts "Adding events to database..."
 		events_done = 0
```

Comment:
You can avoid this counter by using `events.each_with_index do |tba_competition, index|` and index will have the counter value